tasks/db_deploy/test/manual_tests/README.md

```diff
@@ -494,9 +522,26 @@ To run the migration test, in your **python** window run the command
 {"message": "granules table created.", "timestamp": "2021-09-14T13:37:54.711834", "level": "info"}
 {"message": "Creating files table ...", "timestamp": "2021-09-14T13:37:54.711834", "level": "debug"}
 {"message": "files table created.", "timestamp": "2021-09-14T13:37:54.927886", "level": "info"}
-{"message": "Data added to the schema_versions table.", "timestamp": "2021-09-14T15:20:20.231299", "level": "info"}
 {"message": "Populating the schema_versions table with data ...", "timestamp": "2021-09-14T15:20:20.199998", "level": "debug"}
-{"message": "Manual test complete.", "timestamp": "2021-09-14T15:20:20.363841", "level": "info"}
+{"message": "Data added to the schema_versions table.", "timestamp": "2021-09-14T15:20:20.231299", "level": "info"}
+{"message": "Creating reconcile_status table ...", "timestamp": "2022-02-10T10:51:07.421373", "level": "debug"}
+{"message": "reconcile_status table created.", "timestamp": "2022-02-10T10:51:07.422447", "level": "info"}
+{"message": "Creating reconcile_job table ...", "timestamp": "2022-02-10T10:51:07.422552", "level": "debug"}
+{"message": "reconcile_job table created.", "timestamp": "2022-02-10T10:51:07.422962", "level": "info"}
+{"message": "Creating reconcile_s3_object table ...", "timestamp": "2022-02-10T10:51:07.423052", "level": "debug"}
+{"message": "reconcile_s3_object table created.", "timestamp": "2022-02-10T10:51:07.423450", "level": "info"}
+{"message": "Creating reconcile_catalog_mismatch_report table ...", "timestamp": "2022-02-10T10:51:07.423539", "level": "debug"}
+{"message": "reconcile_catalog_mismatch_report table created.", "timestamp": "2022-02-10T10:51:07.424753", "level": "info"}
+{"message": "Creating reconcile_orphan_report table ...", "timestamp": "2022-02-10T10:51:07.424846", "level": "debug"}
+{"message": "reconcile_orphan_report table created.", "timestamp": "2022-02-10T10:51:07.425242", "level": "info"}
+{"message": "Creating reconcile_phantom_report table ...", "timestamp": "2022-02-10T10:51:07.425330", "level": "debug"}
+{"message": "reconcile_phantom_report table created.", "timestamp": "2022-02-10T10:51:07.425728", "level": "info"}
+{"message": "Creating partition table for reconcile_s3_object ...", "timestamp": "2022-02-10T10:51:07.425815", "level": "debug"}
+{"message": "partition table for reconcile_s3_object created.", "timestamp": "2022-02-10T10:51:07.426223", "level": "info"}
```

Comment:
May want to update this message to provide the partition name. This should also occur after the creation of the base table.

Reply:
Fixed